Datastore/src/Connection/Rest.php

```diff
@@ -111,4 +110,25 @@ public function runQuery(array $args)
     {
         return $this->send('projects', 'runQuery', $args);
     }
+
+    /**
+     * @param array $args
+     */
+    public function runAggregationQuery(array $args)
+    {
+        if (isset($args['aggregationQuery']['aggregations'])) {
+            foreach ($args['aggregationQuery']['aggregations'] as &$aggregation) {
+                array_walk(
```

Comment:
I think `array_map` would look better than `array_walk` and passing by reference here:
```php
$aggregation = array_map(
    function ($item) {
        // if empty, force request to encode as {} rather than [].
        return is_array($item) && count($item) === 0) ? new stdClass() : $item
    }, 
    $aggregation
);
```